project/plugins.sbt

```diff
@@ -1,7 +1,7 @@
 val scalaJSVersion =
-  Option(System.getenv("SCALAJS_VERSION")).filter(_ != "").getOrElse("1.0.0")
+  Option(System.getenv("SCALAJS_VERSION")).filter(_ != "").getOrElse("1.0.1")
 
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % scalaJSVersion)
-addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "0.6.0")
+addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.0.0")
 
-addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.3.0")
+addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.8.1")
```

Comment:
It seems the MiMa upgrade kills Scala 2.10.

Reply:
For the record, do we want to keep Scala 2.10?

Reply:
Is there any reason to use the new sbt?